native-schema-registry/csharp/AWSGsrSerDe/AWSGsrSerDe/deserializer/avro/AvroDeserializer.cs

```diff
@@ -71,7 +83,6 @@ private void Configure(GlueSchemaRegistryConfiguration configs)
         {
             _avroRecordType = configs.AvroRecordType;
```

Comment:
Can the code inside `Configure` method be part of the constructor itself? Any advantage of separate configure method?

Reply:
good catch, moved.